src/app/messages/messages.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MessagesComponent } from './messages.component';
+
+describe('MessagesComponent', () => {
+  let component: MessagesComponent;
+  let fixture: ComponentFixture<MessagesComponent>;
+  let h2: HTMLElement;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ MessagesComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(MessagesComponent);
+    component = fixture.componentInstance;
+    h2 = fixture.nativeElement.querySelector('h2');
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should display message title', () => {
+    fixture.detectChanges();
+    expect(h2.textContent).toContain("Messages");
+  });
+
+  it('should display a different message title', () => {
+    h2.textContent = 'Oodles of Noodles';
+    fixture.detectChanges();
+    expect(h2.textContent).toContain('Oodles of Noodles');
```

Comment:
what are you testing here? Seems to me that you are testing setting textContent works as expected - which is not your code.

Reply:
Ah, whoops. I was just learning the syntax. Didn't mean to leave that in.